universal-login-react/src/playground/App.tsx

```diff
@@ -100,6 +101,18 @@ export const App = () => {
                     hideModal={() => history.push('/')}
                   />}
               />
+              <Route
+                exact
+                path="/onboardingEmail"
+                render={({history}) =>
+                  <EmailFlowChooserScreen
```

Comment:
We are not using most of these props. I think that we should just remove them and add them as soon as we need them.